assets/src/design-system/components/contextMenu/menu-item.js

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { useMemo } from 'react';
+import styled, { css } from 'styled-components';
+/**
+ * Internal dependencies
+ */
+import { Button } from '../button';
+
+/**
+ * Internal dependencies
+ */
+import { Link } from '../typography/link';
+import { Text } from '../typography/text';
+
+const ItemText = styled(Text)`
+  width: 200px;
+  text-align: left;
+`;
+
+const Shortcut = styled(Text)(
+  ({ theme }) => css`
+    margin-right: 5px;
+    color: ${theme.colors.bg.tertiary};
+  `
+);
+
+export const MenuItem = ({ disabled, href, label, onClick, shortcut }) => {
+  const textContent = useMemo(
+    () => (
+      <>
+        <ItemText forwardedAs="span">{label}</ItemText>
+        {shortcut && <Shortcut forwardedAs="span">{shortcut}</Shortcut>}
```

Comment:
I think for RTL we may want to switch these, no?

Reply:
Added LTR styling:
<image src="https://user-images.githubusercontent.com/22185279/107672116-cd73c800-6c51-11eb-9b77-4145b5c0c74c.gif" width="300px" />